ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -27,21 +27,65 @@
     {% include "includes/gtm/gtm_head.html" %}
   </head>
   <body>
+    <script src="{% static 'js/src/nav_script.js' %}"></script>
+
     <script>
       document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');
     </script>
     {% include "includes/gtm/gtm_body.html" %}
     {% include "includes/cookie_consent/cookie_banner.html" %}
     <a id="skip-to-main-content" href="#main-content">{% translate "skiplink" %}</a>
     {% include "includes/environment_banner.html" %}
-    {% include "includes/phase_banner.html" %}
+    {% flag beta_homepage %}
+      {% include "includes/phase_banner.html" %}
+    {% endflag %}
     <header class="page-header">
       {% block header %}
-        <div class="page-header__flex-container container py-1">
-          <div class="page-header__nav">
-            <div class="page-header__site-logo">
-              <a href="{% url "home" %}" id="home-link">{% translate "home" %}</a>
-            </div>
+        <div class="page-header__flex-container container py-2">
+          <div class="beta-header">
+            <header class="govuk-header__beta-phase" data-module="govuk-header">
+              <div class="govuk-header__container govuk-width-container">
+                <div class="govuk-header__logo">
+                  <a href="https://www.nationalarchives.gov.uk" class="govuk-header__link govuk-header__link--homepage">
+                    <svg xml:space="preserve" class="beta-header__tna-logo" style="enable-background:new 0 0 160 160" viewBox="0 0 160 160" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                      <path fill="#000" d="M0 0h160v160H0z" class="tna-logo__background"/>
+                      <g class="tna-logo__foreground" fill="#fff">
```

Comment:
If we make `fill="#fff"` to `fill="currentColor"` we can control it via CSS (the CSS already makes it white, but we could also style the hover affect if we wanted to)